scripts/metrics/test_metrics_xml_to_csv.py

```diff
@@ -0,0 +1,198 @@
+"""
```

Comment:
Given that there will be multiple types/formats of files to parse, I assume separate scripts will handle other files, and suggest this is a `ctest_xml_to_csv` script.

Reply:
I haven't made the decision yet to split the metrics script. I will rename the file according when that's decided